### PR TITLE
Added output in DOT format for class hierarchy.

### PR DESCRIPTION
**Pull request by marek-trtik**

### CODE REVIEW

File: src/goto-programs/class_hierarchy.cpp [hunk 1]
```diff
@@ -135,3 +135,18 @@ void class_hierarchyt::output(std::ostream &out) const
           << ch << '\n';
   }
 }
+
+std::ostream &output_dot(std::ostream &ostr, const class_hierarchyt &hierarchy)
```
Comment (peterschrammel) [inline src/goto-programs/class_hierarchy.cpp]: function comment block missing

File: src/goto-programs/class_hierarchy.h [hunk 2]
```diff
@@ -55,4 +55,19 @@ class class_hierarchyt
   void get_parents_trans_rec(const irep_idt &, idst &) const;
 };
 
+
+/*******************************************************************\
```
Comment (peterschrammel) [inline src/goto-programs/class_hierarchy.h]: no function comment block here

**Comment by marek-trtik** [comment]
Requested updates are available.

File: src/goto-programs/class_hierarchy.h [hunk 3]
```diff
@@ -55,4 +55,8 @@ class class_hierarchyt
   void get_parents_trans_rec(const irep_idt &, idst &) const;
 };
 
+
```
Comment (peterschrammel) [inline src/goto-programs/class_hierarchy.h]: remove one blank line

File: src/goto-programs/class_hierarchy.h [hunk 4]
```diff
@@ -55,4 +55,8 @@ class class_hierarchyt
...
+std::ostream &output_dot(std::ostream &ostr, const class_hierarchyt &hierarchy);
+
```
Comment (peterschrammel) [inline src/goto-programs/class_hierarchy.h]: remove one blank line

**Comment by marek-trtik** [comment]
Requested updates are available.

**Pull request merged**